src/libs/actions/User.js

```diff
@@ -558,15 +566,29 @@ function subscribeToUserEvents() {
             // until we finish the migration to reliable updates. So let's check it before actually updating
             // the properties in Onyx
             if (pushJSON.lastUpdateID && pushJSON.previousUpdateID) {
-                console.debug('[OnyxUpdates] Received lastUpdateID from pusher', pushJSON.lastUpdateID);
-                console.debug('[OnyxUpdates] Received previousUpdateID from pusher', pushJSON.previousUpdateID);
-                // Store these values in Onyx to allow App.reconnectApp() to fetch incremental updates from the server when a previous session is being reconnected to.
-                Onyx.multiSet({
-                    [ONYXKEYS.ONYX_UPDATES.LAST_UPDATE_ID]: Number(pushJSON.lastUpdateID || 0),
-                    [ONYXKEYS.ONYX_UPDATES.PREVIOUS_UPDATE_ID]: Number(pushJSON.previousUpdateID || 0),
-                });
+                const pushJSONLastUpdateID = Number(pushJSON.lastUpdateID || 0);
+                const pushJSONPreviousUpdateID = Number(pushJSON.previousUpdateID || 0);
+
+                console.debug('[OnyxUpdates] Received lastUpdateID from pusher', pushJSONLastUpdateID);
+                console.debug('[OnyxUpdates] Received previousUpdateID from pusher', pushJSONPreviousUpdateID);
+                console.debug('[OnyxUpdates] The lastUpdateID the client received was', onyxUpdatesLastUpdateID);
+
```

Comment:
```suggestion
                console.debug('[OnyxUpdates] The lastUpdateID the client has is', onyxUpdatesLastUpdateID);

```

Just for clarity, I was confused if this was the one I received or the one we had when just looking at the logs

Reply:
I will update!